easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
@rjeschmi: not 100% sure what you mean, but I think the answer is no (currently). I do see a use for it though, e.g.:

```
name = 'hanythingondemand'
...
module_name = 'hod'
```